mito-ai/src/icons/RobotHeadIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const RobotHeadIcon: React.FC<{
```

Comment:
I think in practice the robot icon might be a little too like detailed for the size we are displaying it at? 
<img width="160" alt="Screenshot 2025-02-07 at 1 06 05 PM" src="https://github.com/user-attachments/assets/19c2bd0b-b7a4-4123-9f00-7b0b760895b3" />

Maybe we can find a slightly less detailed icon and one that is more symmetrical in height and width. it looks a bit strange since they take up different amounts of space naturally